PROJECT

```diff
@@ -1,16 +1,49 @@
 version: "3"
-domain: x-k8s.io
+domain: cluster.x-k8s.io
 repo: sigs.k8s.io/cluster-api-provider-azure
 resources:
-- group: infrastructure
-  version: v1beta1
-  kind: AzureMachine
-- group: infrastructure
-  version: v1beta1
-  kind: AzureCluster
-- group: infrastructure
-  version: v1beta1
-  kind: AzureMachineTemplate
-- group: infrastructure
-  version: v1beta1
-  kind: AzureClusterTemplate
+# -----------------
+# v1beta1 types
+# -----------------
+  - group: infrastructure
+    version: v1beta1
+    kind: AzureCluster # present
```

Comment:
Are all these `# present` comments needed? I always worry about how YAML processors will actually handle comments, and I don't know what they mean.

Reply:
Sorry about that! That was me cross-checking if a particular GVK was present in CAPZ.

Reply:
Updated, please take a look again.